TileFormats/Instanced3DModel/README.md

```diff
@@ -92,70 +123,64 @@ These semantics define global properties for all instances.
-
+A box in the standard basis:
 ![box standard basis](figures/box-standard-basis.png)
 
-**Figure 3**: A box transformed into a rotated basis.
-
+A box transformed into a rotated basis
 ![box rotated basis](figures/box-rotated-basis.png)
 
-#### Oct-encoded Normal Vectors
+#### Oct-encoded normal vectors
 
 If `NORMAL_UP` and `NORMAL_RIGHT` are not defined for an instance, its orientation may be stored as oct-encoded normals in `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P`.
-These define `up` and `right` using the oct-encoding described in
-[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/).
-An implementation for encoding and decoding these unit vectors can be found in Cesium's
-[AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+These define `up` and `right` using the oct-encoding described in [*A Survey of Efficient Representations of Independent Unit Vectors*](http://jcgt.org/published/0003/02/01/). Oct-encoded values are stored in unsigned, unnormalized range (`[0, 65535]`) and then mapped to a signed normalized range (`[-1.0, 1.0]`) at runtime.
+
+> An implementation for encoding and decoding these unit vectors can be found in Cesium's [AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
 module.
 
-#### Default Orientation
+#### Default orientation
 
-If `NORMAL_UP` and `NORMAL_RIGHT` or `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P` are not present,
-the instance will not have a custom orientation. If `EAST_NORTH_UP` is `true` the instance is assumed to be on the `WGS84` ellipsoid and its orientation will default to the `east/north/up` reference frame at its Cartographic position.
-This is suitable for instanced models like trees whose orientation is always facing up from their position on the ellipsoid's surface.
+If `NORMAL_UP` and `NORMAL_RIGHT` or `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P` are not present, the instance will not have a custom orientation. If `EAST_NORTH_UP` is `true`, the instance is assumed to be on the `WGS84` ellipsoid and its orientation will default to the `east/north/up` reference frame at its cartographic position.
+This is suitable for instanced models such as trees whose orientation is always facing up from 
 
-### Instance Position
+### Instance position
 
-`POSITION` defines the location for an instance before any tile transforms are applied. Positions may be defined relative-to-center for high-precision rendering [4]. `RTC_CENTER` defines the center position.
+`POSITION` defines the location for an instance before any tile transforms are applied. Positions may be defined relative to center for high-precision rendering, see [Precisions, Precisions](http://blogs.agi.com/insight3d/index.php/2008/09/03/precisions-precisions/). `RTC_CENTER` defines the center position.
```

Comment:
The blog link doesn't seem to exist anymore. Does anyone know if this blog is still live anywhere? In any case this should just stay as a footnote. (Same comments for pnts)

Reply:
Actually I see that throughout foot notes were removed, so fine to keep inline once the link is fixed.